query-engine/connectors/mongodb-query-connector/src/root_queries/write.rs

```diff
@@ -374,3 +375,40 @@ pub async fn m2m_disconnect<'conn>(
 
     Ok(())
 }
+
+// TODO: We always return 0 for now. MongoDB doesn't have a baked-in API to get the numer of affected rows like SQL dbs
+// TODO: This is temporary until we decide what to do https://github.com/prisma/prisma-engines/issues/2391
```

Comment:
Not sure about this one, since it looks like update command can indeed return the number of the affected documents.

Mongo docs: https://docs.mongodb.com/manual/reference/command/update/#returns

Reply:
Hey, you're correct but the comment is also still correct. There's no baked-in API in the driver to get the number of affected rows. Returning that number would imply manually parsing the responses, which we might do, but we wanna postpone that a bit